roofit/roofitcore/inc/RooAbsReal.h

```diff
@@ -84,18 +83,19 @@ class RooAbsReal : public RooAbsArg {
   /// the variables to normalise over.
   /// These are integrated over their current ranges to compute the normalisation constant,
   /// and the unnormalised result is divided by this value.
+  inline Double_t getVal(const RooArgSet* normalisationSet = nullptr) const {
 #ifdef ROOFIT_CHECK_CACHED_VALUES
-  Double_t getVal(const RooArgSet* normalisationSet = nullptr) const;
+    return _DEBUG_getVal(normalisationSet);
```

Comment:
Please note that technically the name with a leading underscore are reserved by the compiler ...